cronenbroguelike/game.py

```diff
@@ -103,6 +103,10 @@ def main():
     game_config = _load_config()
     if game_config.get("extra_commands"):
         from . import extra_commands
-    _start_game(None, game_config)
-    adventurelib.say("")  # Necessary for space before first prompt.
-    adventurelib.start()
+    while True:
+        try:
+            _start_game(None, game_config)
+            adventurelib.say("")  # Necessary for space before first prompt.
+            adventurelib.start()
+        except tartarus.RaptureException:
+            continue
```

Comment:
Is `continue` materially different than `pass` here?

Reply:
Nope! I suppose `pass` works just as well.